fpConfig.cmake.in

```diff
@@ -0,0 +1,13 @@
+# - Config file for the fp package
+# It defines the following variables
+#  fp_INCLUDE_DIRS - include directories for FooBar
+#  fp_LIBRARIES    - libraries to link against
+#  fp_EXECUTABLE   - the bar executable
+
+set(fp_INCLUDE_DIRS "@CONF_INCLUDE_DIRS@")
+
+# Our library dependencies (contains definitions for IMPORTED targets)
+include("${fp_DIR}/export_fp.cmake")
+
+# These are IMPORTED targets created by fpTargets.cmake
+set(fp_LIBRARIES fp)
```

Comment:
I wouldn't add this because I would never recommend anyone link to `fp` through any other means than linking to the CMake target. Same goes for `INCLUDE_DIRS`. That's not something you want users relying on so don't give it to them in the first place.

Reply:
This I'm most unsure about because this comes from `ros2 pkg create` so I don't know if this is a shim for ros cmake stuff.

Reply:
How easily can we test if that is true?